CaloFilters/fcl/prolog_trigger.fcl

```diff
@@ -118,6 +118,15 @@ CaloFilters : {
 	triggerPath             : "CaloPhoton"	    
     }
   
+    caloNNFilter : {
+	module_type            : FilterEcalNNTrigger
+	caloClusterCollection  : CaloClusterFast
+        caloBkgMVA             :{ MVAWeights : "CaloFilters/data/CE_NN_ReLU.weights.xml" }
+        minEtoTest             : 40
+        minMVAScore            : 0.95
+        trigPath               : "CaloNewClusterTrigger"
+        diagLevel              : 0
+    }
```

Comment:
at the bottom of this file, you should define the sequence to run this filter. This means that you need to instantiate also its dedicated event-prescaler module.

Reply:
I think I did this correctly. If not, I'll let you takeover and do the right thing.